src/circuit/operator/apply.rs

```diff
@@ -37,3 +37,31 @@ where
         (self.func)(i1)
     }
 }
+
+/// Converts any type to a string
+pub struct Stringifier {}
+
+impl Stringifier {
+    #[allow(dead_code)]
```

Comment:
You need to re-export this operator from `circuit/operator/mod.rs`. Then this won't be dead code. Same for `Differentiator`.